core/jazz_acl/config/global-config.json

```diff
@@ -1,4 +1,27 @@
 {
+  "CASBIN": {
```

Comment:
These can be environment specific. They should be living in $env-config.json.

Reply:
Yes. But I saw we are configuring one DB and loading it for all the environments. That's why I changed that. Please see the configuration
https://github.com/tmobile/jazz/blob/master/builds/jazz-build-module/service-configuration-data-loader.groovy#L62

Reply:
Changed CASBIN as environment specific